README.md

```diff
@@ -21,20 +25,40 @@ Run:
 ```
 omero certificates
 ```
+```
+OpenSSL 1.1.1d  10 Sep 2019
+Generating RSA private key, 2048 bit long modulus (2 primes)
+.+++++
+.............................+++++
+e is 65537 (0x010001)
+certificates created: /OMERO/certs/server.key /OMERO/certs/server.pem /OMERO/certs/server.p12
+```
 to update your OMERO.server configuration and to generate or update your self-signed certificates.
 If you already have the necessary configuration settings this plugin will not modify them, so it is safe to always run `omero certificates` every time you start OMERO.server.
 You can now start your omero server as normal.
 
-This plugin automatically overrides the defaults for the following properties if they're not explcitly set:
+This plugin automatically overrides the defaults for the following properties if they're not explicitly set:
 - `omero.glacier2.IceSSL.Ciphers=HIGH`: the default weaker ciphers may not be supported on some systems
 - `omero.glacier2.IceSSL.ProtocolVersionMax=TLS1_2`: Support TLS 1.1 and 1.2, not just 1.0
 - `omero.glacier2.IceSSL.Protocols=TLS1_0,TLS1_1,TLS1_2`: Support TLS 1.1 and 1.2, not just 1.0
 
 The original values can be found on https://docs.openmicroscopy.org/omero/5.6.0/sysadmins/config.html#glacier2
 
+Certificates will be stored under `{omero.data.dir}/certs` by default.
+Set `omero.glacier2.IceSSL.DefaultDir` to change this.
+
 If you see a warning message such as
 ```
 Can't load ./.rnd into RNG
 ```
 it should be safe to ignore.
-Check that you have a `*.p12` certificate, if you are using the default OMERO confioguration this will be in `/OMERO/certs/server.p12`.
+
+For full information see the output of:
+```
+omero certificates --help
+```
+
+## Developer notes
+
+This project uses [setuptools-scm](https://pypi.org/project/setuptools-scm/).
```

Comment:
Two points:
 * This prevents the use of `pip install https://.../archive/master.zip`:
```
    LookupError: setuptools-scm was unable to detect version for '/private/var/folders/z5/txc_jj6x5l5cm81r56ck1n9c0000gn/T/pip-req-build-73o_5uwq'.

    Make sure you're either building from a fully intact git repository or PyPI tarballs. Most other sources (such as GitHub's tarballs, a git checkout without the .git folder) don't contain the necessary metadata and will not work.

    For example, if you're using pip, instead of https://github.com/user/proj/archive/master.zip use git+https://github.com/user/proj.git#egg=proj
```
 * It'd be good if we could unify this across this projects.